app/presenters/metrics_presenter.rb

```diff
@@ -96,6 +96,16 @@ def delivery_organisations_count
     end
   end
 
+  def visible_services_count
+    metric_groups.drop(1).sum { |m|
```

Comment:
Is this `drop(1)` here because the first one is the "totals" metric group?

Reply:
Yes.

Reply:
Perhaps `metric_groups.without(totals_metric_group_presenter)` would be clearer?

[`Enumerable#without`](http://api.rubyonrails.org/classes/Enumerable.html#method-i-without)